docs/changes.xml

```diff
@@ -31,6 +31,12 @@ NGINX Unit updated to 1.27.0.
          date="" time=""
          packager="Andrei Belov &lt;defan@nginx.com&gt;">
 
+<change type="feature">
+<para>
+support $ sub-delimiters in URIs by using "$$".
```

Comment:
I'm not sure if we should add it to the changelog as adding support for $ in URIs, or if we should add it as supporting literal $ after variable resolution with $$.  Opinions?  Cc: @artemkonev

Reply:
Yes, I think you're right as this is the more general case with $ sub-delimiters being a more specific use of this. I'll also update the commit subject...